src/uwtools/nice_dict.py

```diff
@@ -0,0 +1,173 @@
+# (C) Copyright 2020-2022 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Part of this software is developed by the Joint Center for Satellite Data Assimilation (JCSDA) together with its partners.
+#
+# (C) Copyright 2022-2022 NOAA
```

Comment:
Can we refer and table this?

Reply:
I think it could potentially be important to resolve it before we publish it as our code. @aerorahul ?

Reply:
@aerorahul Do you have any thoughts here?

Reply:
Are we using a NOAA copyright on anything else we contribute here?

Reply:
No, we weren't until these JCSDA files were being introduced.  I leave it to the PO to decide the path forward.

Reply:
Why do we these external libraries?  I don't see the benefit but cost is adding more unneeded context for things the language already does by itself and creating a greater barrier to entry for onboarding developers.  Instead of needing to know Python, now you need to know Python, Solo, NiceDict, etc. etc.